### PR TITLE
feat: Add guide for multisig address

### PR DESCRIPTION
**Pull request by yanguoyu**
Refer to https://github.com/Magickbase/neuron-public-issues/issues/401

https://github.com/nervosnetwork/neuron/assets/12881040/3cbc69b8-df5b-4b17-8640-efaaa32e145b

### CODE REVIEW

**Comment by Danie0918** [comment]
@twhy @homura @devchenyan  Please have a review.

File: packages/neuron-ui/src/locales/en.json [hunk 1]
```diff
@@ -1083,6 +1083,8 @@
     },
     "multisig-address": {
       "window-title": "Multisig Addresses",
+      "guide-tip": "For first time users you can check the documentation to <0>learn More</0>.",
```
Comment (homura) [inline packages/neuron-ui/src/locales/en.json:1086]: `Refer to the Multisig documentation to <0>learn more</0>` is fine

**Review by homura — APPROVED** [review]

**Review by twhy — APPROVED** [review]

**Comment by silySuper** [comment]
https://github.com/nervosnetwork/neuron/actions/runs/9638580208 this package can not save lanuage as setting correctly.so some language can not see.

https://github.com/nervosnetwork/neuron/assets/25971273/870a44b1-e8ed-4670-8cca-4b9ebce4cefd

**Comment by yanguoyu** [comment]
> https://github.com/nervosnetwork/neuron/actions/runs/9638580208此包无法按正确设置保存语言，因此有些语言无法看到。
> 
>  2024-06-24.16.52.21.mov

I guess the language will take effect after restart the Neuron

**Comment by Keith-CY** [comment]
> > [nervosnetwork/neuron/actions/runs/9638580208](https://github.com/nervosnetwork/neuron/actions/runs/9638580208) this package can not save lanuage as setting correctly.so some language can not see.
> > 2024-06-24.16.52.21.mov
> 
> I guess the language will take effect after restart the Neuron

Only Chinese/Chinese Simplified are not rendered instantly, other languages are switched as expected.

**Comment by silySuper** [comment]
> > https://github.com/nervosnetwork/neuron/actions/runs/9638580208 this package can not save lanuage as setting correctly.so some language can not see.
> > 2024-06-24.16.52.21.mov
> 
> I guess the language will take effect after restart the Neuron

I found that when I change language and click apply ,the setting can not change.If it changed,language will effect.Does the language  change data saved correctly in database after clicking apply button?When restart,it changed.

**Comment by silySuper** [comment]
https://github.com/orgs/Magickbase/projects/6/views/2?pane=issue&itemId=68446485

**Review by silySuper — APPROVED** [review]

**Pull request merged**